### PR TITLE
User report page cpu barchart 4130

### PR DESCRIPTION
**Pull request by inuwa**
## Done

Updated the barchart for the user report page What CPU section to match the design documentation.
## QA

- Check out this feature branch user 
- Run the site using the command `./run serve`
- View the site locally in your web browser at: [http://0.0.0.0:8001/](http://0.0.0.0:8001/)
- Run through the following [QA steps](https://github.com/canonical-webteam/practices/blob/master/workflow/qa-steps.md)
- go to the url http://0.0.0.0:8001/desktop/statistics
- Check that the layout matches the design https://github.com/canonical-websites/www.ubuntu.com/issues/4094

## Issue / Card

Fixes https://github.com/canonical-websites/www.ubuntu.com/issues/4130

## Screenshots

image

### CODE REVIEW

File: static/js/desktopStatistics.js [hunk 1]
```diff
@@ -620,8 +619,24 @@ function buildCharts() {
       title: "VMs"
     }
   );
-  createBarChart('#number-of-cpus', dummyData.cpus.dataset);
-  createBarChart('#size-of-ram', dummyData.ram.dataset);
+  createBarChart('#number-of-cpus', dummyData.cpus.dataset, {
+    colors: ['#ed764d', '#925375'],
```
Comment (anthonydillon) [inline static/js/desktopStatistics.js]: Can you use '#E95420', '#772953' instead.

Comment (kwm14) [reply static/js/desktopStatistics.js]: @anthonydillon just spotted that too 👍

**Comment by kwm14** [comment]
@inuwa LGTM 👍 just need to update color hex values in @anthonydillon comment above and then we're good to go 💯

**Comment by inuwa** [comment]
@kwm14 and @anthonydillon thank you for the spot. I made the corrections now.

**Comment by kwm14** [comment]
@inuwa perfect, colours look good 👍

**Pull request merged**